### ISSUE TITLE
Builder ignores defaultExtension configuration

### ISSUE THREAD

**Issue by maxlk** [systemjs/builder#460]
After upgrade from *jspm 0.16.19* to *0.16.20* (from *systemjs-builder 0.14.15* to *0.15.0*) the builder started to append `.js` extension despite `defaultExtension: false` is specified.

**Example:**
In `config.js` the following package is configured:
``` javascript
packages: {
  'test-app': {
    main: 'main.jsx',
    format: 'esm',
    defaultExtension: false
  }
}
```

When launching build with `jspm bundle-sfx test-app ./bundle.js` the following error occurs:
```
     Building the single-file sfx bundle for test-app...

err  Error on fetch for test-app/main.jsx at file:///C:/test-project/test-app/main.jsx.js
        Error: ENOENT, open 'C:\test-project\test-app\main.jsx.js'
         at Error (native)
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/systemjs/commit/6b3d426c07cd6b913c0db392c03784e28c5c8929.

**Comment by guybedford** [comment]
Released in 0.15.2.

**Comment by maxlk** [comment]
Great! Thanks!
